src/problem.rs

```diff
@@ -20,7 +20,7 @@ const QUESTION_QUERY_OPERATION: &str = "questionData";
 pub fn get_problem(id: u32) -> Option<Problem> {
     let problems = get_problems().unwrap();
     for problem in problems.stat_status_pairs.iter() {
-        if problem.stat.question_id == id {
+        if problem.stat.frontend_question_id == id {
```

Comment:
Is this just for `leetcode-cn.com` or general case?

Reply:
General case. Tested with leetcode.com